ci/eval/compare/maintainers.nix

```diff
@@ -82,7 +82,7 @@ let
   listToPing = lib.concatMap (
     pkg:
     builtins.map (maintainer: {
-      id = maintainer.githubId;
+      id = if maintainer ? githubId then maintainer.githubId else "unknown";
```

Comment:
Aside from the fact that you shouldn't modify CI files without considering how it will impact other maintainers, this change still doesn't fix the CI issue, so please revert it.

Reply:
Note that changes to how CI runs won't be reflected in the PR that makes those changes. One needs to make a PR against the PR's branch to test whether this works.